README.md

```
@@ -88,7 +88,7 @@ $ npm run site-serve
 
 コードのチェック
 ```bash
-$ # eslint, tsc --noEmit
+$ # Biome, tsc --noEmit, secretlint
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)